curriculum/challenges/english/07-scientific-computing-with-python/build-an-arithmetic-formatter-project/arithmetic-formatter.md

```diff
@@ -68,6 +68,8 @@ The function will return the correct conversion if the supplied problems are pro
   - There should be four spaces between each problem.
   - There should be dashes at the bottom of each problem. The dashes should run along the entire length of each problem individually. (The example above shows what this should look like.)
 
+Note: open the browser console with F12 to see a more verbose output of the tests.
```

Comment:
The sentence can be slightly modified but I would add something similar to each certification project.